### ISSUE TITLE
[Forge: 1.19.2 - 0.2.3] Common Config problem (breaks all vanilla loot tables)

### ISSUE THREAD

**Issue by SiverDX** [AstralOrdana/Spelunkery#92]
Updated the mod and the log gets spammed (over 200k lines) with this (+ for every parse of a loot table)
I already deleted the old config files and tried creating a new world

The game itself... works? Maybe? Not sure, it loads and I can do stuff but I haven't let it run for too long

``` log
[23Jul2023 23:50:31.020] [Render thread/ERROR] [net.minecraftforge.eventbus.EventBus/EVENTBUS]: Exception caught during firing event: Cannot invoke "java.util.function.Supplier.get()" because "com.ordana.spelunkery.configs.CommonConfigs.ORE_STONE_DROPS" is null
	Index: 28
	Listeners:
		0: HIGHEST
		1: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@1e339c0c
		2: HIGH
		3: ASM: class dev.architectury.event.forge.EventHandlerImplCommon event(Lnet/minecraftforge/event/LootTableLoadEvent;)V
		4: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@3a328f62
		5: NORMAL
		6: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@1d0e76e9
		7: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@5715cf54
		8: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@55c4ea46
		9: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@7ba33603
		10: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@1574bea0
		11: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@6ce2bc35
		12: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@1e06ed35
		13: ASM: class com.mlib.gamemodifiers.contexts.OnLootTableLoad onLootTableLoad(Lnet/minecraftforge/event/LootTableLoadEvent;)V
		14: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@7c77c3d8
		15: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@4c46e348
		16: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@35bcfde1
		17: ASM: class com.teammetallurgy.aquaculture.init.AquaLootTables onLootTableLoad(Lnet/minecraftforge/event/LootTableLoadEvent;)V
		18: ASM: com.github.alexthe668.domesticationinnovation.client.ClientProxy@7f8172f1 onLootTableLoad(Lnet/minecraftforge/event/LootTableLoadEvent;)V
		19: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@1c7ac126
		20: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@5e51a872
		21: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@25765c53
		22: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@7777fee9
		23: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@3dce72d0
		24: ASM: class by.dragonsurvivalteam.dragonsurvival.server.handlers.LootTableInject lootLoad(Lnet/minecraftforge/event/LootTableLoadEvent;)V
		25: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@6e0f4f23
		26: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@790c1275
		27: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@31443b0b
		28: ASM: class com.ordana.spelunkery.forge.ForgeEvents onAddLootTables(Lnet/minecraftforge/event/LootTableLoadEvent;)V
		29: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@7fd29701
		30: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@6bb0a98f
		31: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@72b69136
		32: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@623a6008
		33: ASM: class net.mehvahdjukaar.supplementaries.common.events.forge.ServerEventsForge onAddLootTables(Lnet/minecraftforge/event/LootTableLoadEvent;)V
		34: ASM: vazkii.quark.content.tools.module.ColorRunesModule@64e7e24b onLootTableLoad(Lnet/minecraftforge/event/LootTableLoadEvent;)V
		35: ASM: vazkii.quark.content.tools.module.EndermoshMusicDiscModule@575b76e onLootTableLoad(Lnet/minecraftforge/event/LootTableLoadEvent;)V
		36: ASM: vazkii.quark.content.tools.module.AncientTomesModule@daa2c2f onLootTableLoad(Lnet/minecraftforge/event/LootTableLoadEvent;)V
		37: ASM: vazkii.quark.content.world.module.AncientWoodModule@307bd35a onLootTableLoad(Lnet/minecraftforge/event/LootTableLoadEvent;)V
		38: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@93b99d1
		39: LOW
		40: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@67a9b5a8
		41: LOWEST
		42: net.minecraftforge.eventbus.EventBus$$Lambda$1672/0x00000008013b5000@2150a74a
		43: ASM: com.aizistral.enigmaticlegacy.handlers.EnigmaticEventHandler@e5a04e onLootTablesLoaded(Lnet/minecraftforge/event/LootTableLoadEvent;)V
		44: ASM: com.aizistral.etherium.core.EtheriumEventHandler@1bf824f6 onLootTablesLoaded(Lnet/minecraftforge/event/LootTableLoadEvent;)V
java.lang.NullPointerException: Cannot invoke "java.util.function.Supplier.get()" because "com.ordana.spelunkery.configs.CommonConfigs.ORE_STONE_DROPS" is null
	at TRANSFORMER/spelunkery@1.19.2-0.2.3/com.ordana.spelunkery.events.ModLootInjects.onLootInject(ModLootInjects.java:114)
	at TRANSFORMER/spelunkery@1.19.2-0.2.3/com.ordana.spelunkery.forge.ForgeEvents.onAddLootTables(ForgeEvents.java:25)
	at TRANSFORMER/spelunkery@1.19.2-0.2.3/com.ordana.spelunkery.forge.__ForgeEvents_onAddLootTables_LootTableLoadEvent.invoke(.dynamic)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:315)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:296)
	at TRANSFORMER/forge@43.2.14/net.minecraftforge.event.ForgeEventFactory.loadLootTable(ForgeEventFactory.java:577)
	at TRANSFORMER/forge@43.2.14/net.minecraftforge.common.ForgeHooks.loadLootTable(ForgeHooks.java:940)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.world.level.storage.loot.LootTables.lambda$apply$0(LootTables.java:41)
	at TRANSFORMER/kubejs@1902.6.1-build.327/dev.latvian.mods.kubejs.core.LootTablesKJS.kjs$apply0(LootTablesKJS.java:36)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.world.level.storage.loot.LootTables.redirect$zbc000$kjs$apply(LootTables.java:520)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.world.level.storage.loot.LootTables.m_5787_(LootTables.java:38)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.world.level.storage.loot.LootTables.m_5787_(LootTables.java:16)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.server.packs.resources.SimplePreparableReloadListener.m_10789_(SimplePreparableReloadListener.java:13)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(Unknown Source)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:157)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.util.thread.ReentrantBlockableEventLoop.m_6367_(ReentrantBlockableEventLoop.java:23)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:131)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.util.thread.BlockableEventLoop.m_18699_(BlockableEventLoop.java:116)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1072)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.Minecraft.m_91374_(Minecraft.java:700)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.main.Main.m_239872_(Main.java:212)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.main.Main.main(Main.java:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at MC-BOOTSTRAP/fmlloader@1.19.2-43.2.14/net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.run(Launcher.java:106)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:57)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71)
```

``` log
[23Jul2023 23:54:54.435] [Render thread/ERROR] [net.minecraft.world.level.storage.loot.LootTables/]: Couldn't parse loot table minecraft:blocks/zombie_head
java.lang.NullPointerException: Cannot invoke "java.util.function.Supplier.get()" because "com.ordana.spelunkery.configs.CommonConfigs.ORE_STONE_DROPS" is null
	at com.ordana.spelunkery.events.ModLootInjects.onLootInject(ModLootInjects.java:114) ~[spelunkery-1.19.2-0.2.3-forge.jar%23765!/:?]
	at com.ordana.spelunkery.forge.ForgeEvents.onAddLootTables(ForgeEvents.java:25) ~[spelunkery-1.19.2-0.2.3-forge.jar%23765!/:?]
	at com.ordana.spelunkery.forge.__ForgeEvents_onAddLootTables_LootTableLoadEvent.invoke(.dynamic) ~[spelunkery-1.19.2-0.2.3-forge.jar%23765!/:?]
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.3.jar%23134!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.3.jar%23134!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.3.jar%23134!/:?]
	at net.minecraftforge.event.ForgeEventFactory.loadLootTable(ForgeEventFactory.java:577) ~[forge-1.19.2-43.2.14-universal.jar%23830!/:?]
	at net.minecraftforge.common.ForgeHooks.loadLootTable(ForgeHooks.java:940) ~[forge-1.19.2-43.2.14-universal.jar%23830!/:?]
	at net.minecraft.world.level.storage.loot.LootTables.lambda$apply$0(LootTables.java:41) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at java.util.HashMap.forEach(Unknown Source) ~[?:?]
	at net.minecraft.world.level.storage.loot.LootTables.m_5787_(LootTables.java:38) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.world.level.storage.loot.LootTables.m_5787_(LootTables.java:16) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.server.packs.resources.SimplePreparableReloadListener.m_10789_(SimplePreparableReloadListener.java:13) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source) ~[?:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:157) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.m_6367_(ReentrantBlockableEventLoop.java:23) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:131) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.util.thread.BlockableEventLoop.m_18701_(BlockableEventLoop.java:140) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.Minecraft.m_18701_(Minecraft.java:4419) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.gui.screens.worldselection.WorldOpenFlows.m_233096_(WorldOpenFlows.java:130) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.gui.screens.worldselection.WorldOpenFlows.m_233122_(WorldOpenFlows.java:109) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.gui.screens.worldselection.WorldOpenFlows.doLoadLevel(WorldOpenFlows.java:148) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.gui.screens.worldselection.WorldOpenFlows.m_233145_(WorldOpenFlows.java:136) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.gui.screens.worldselection.WorldOpenFlows.handler$cae000$kiwi$loadLevel(WorldOpenFlows.java:1032) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.gui.screens.worldselection.WorldOpenFlows.m_233133_(WorldOpenFlows.java) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.gui.screens.worldselection.WorldSelectionList$WorldListEntry.m_101744_(WorldSelectionList.java:586) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.gui.screens.worldselection.WorldSelectionList$WorldListEntry.m_101704_(WorldSelectionList.java:448) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.gui.screens.worldselection.WorldSelectionList$WorldListEntry.m_6375_(WorldSelectionList.java:378) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.gui.components.AbstractSelectionList.m_6375_(AbstractSelectionList.java:336) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.gui.components.events.ContainerEventHandler.m_6375_(ContainerEventHandler.java:27) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.MouseHandler.m_168084_(MouseHandler.java:88) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.gui.screens.Screen.m_96579_(Screen.java:522) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.MouseHandler.m_91530_(MouseHandler.java:85) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.MouseHandler.m_168091_(MouseHandler.java:185) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.util.thread.BlockableEventLoop.execute(BlockableEventLoop.java:103) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.MouseHandler.m_91565_(MouseHandler.java:184) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43) ~[lwjgl-glfw-3.3.1.jar%2384!/:build 7]
	at org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.3.1.jar%2396!/:build 7]
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3403) ~[lwjgl-glfw-3.3.1.jar%2384!/:build 7]
	at com.mojang.blaze3d.systems.RenderSystem.m_69495_(RenderSystem.java:204) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at com.mojang.blaze3d.platform.Window.m_85435_(Window.java:337) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1143) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:700) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.main.Main.m_239872_(Main.java:212) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[client-1.19.2-20220805.130853-srg.jar%23825!/:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27) ~[fmlloader-1.19.2-43.2.14.jar%23150!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) [bootstraplauncher-1.1.2.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:57) [ForgeWrapper-mmc2.jar:mmc2]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71) [NewLaunch.jar:?]
```

**Comment by SiverDX** [comment]
Since you're using this config for Forge you also need to define it for Forge:
[ore_stone_drops](https://github.com/AstralOrdana/Spelunkery/blob/8d1eec06a57ed2f45cfeef45e947cab6110c2993/common/src/main/java/com/ordana/spelunkery/configs/CommonConfigs.java#L120C13-L120C29)

**Comment by HolidayTheRaptor** [comment]
I've played around a bit, it seems that vanilla blocks simply don't drop. anything. unless they're ores, any breakable blocks don't drop anything. mining stone yields 0 cobble or anything.

**Comment by AstralOrdana** [comment]
known issue, we're working on getting a fix out ASAP

**Comment by AstralOrdana** [comment]
0.2.4 version released to fix this for 1.19.2, issue persists in 1.20

**Comment by AstralOrdana** [comment]
reopened because i forgot i left it open because the issue persists in 1.20

**Comment by AstralOrdana** [comment]
fixed for 1.20.1-0.3.1
